### DISCUSSION TITLE
Release of Extension DLLs

### DISCUSSION THREAD

**Discussion by kamillo** [apache/logging-log4net#161] [Q&A] [answered]
I would like to ask if there are any plans to include extensions (e.g. log4net.Ext.EventID) in the future releases?

**Comment by FreeAndNil** [comment, marked as answer]
Hi @kamillo,

thanks for your question.
I'm sorry, but we will continue to publish the examples exclusively as source code on Github.
We've recently cleaned them up and moved to .net8 (see https://github.com/apache/logging-log4net/tree/Feature/111-Dropping-support-for-older-runtimes/examples).

Hope this helps.

Jan

**Reply by kamillo** [reply]
Hi @FreeAndNil,

Much appreciated, that clears it up.
